lib/ansible/modules/cloud/misc/proxmox.py

```diff
@@ -428,6 +436,7 @@ def main():
       timeout = dict(type='int', default=30),
       force = dict(type='bool', default='no'),
       state = dict(default='present', choices=['present', 'absent', 'stopped', 'started', 'restarted']),
+      unprivileged = dict(type='int', default=0)
     )
```

Comment:
I wonder if you should have `, choices=['0', '1']` here to protect from invalid values being passed in